release.gradle

```diff
@@ -5,15 +5,15 @@ def isReleaseBuild() {
     return VERSION_NAME.contains("SNAPSHOT") == false
 }
 
-def getRepositoryUrl() {
+def getMavenRepositoryUrl() {
```

Comment:
This error started with gradle 3.4 and originates from hasProperty() returning true even if the property is indeed missing, but still has an overrding getter.
The simpler fix is to change
```
def getRepositoryUrl() {
    return **hasProperty('repositoryUrl')** ? property('repositoryUrl') : 'https://oss.sonatype.org/service/local/staging/deploy/maven2/'
}
```
Into:
```
def getRepositoryUrl() {
    return **findProperty('repositoryUrl') != null** ? property('repositoryUrl') : 'https://oss.sonatype.org/service/local/staging/deploy/maven2/'
}
```
findProperty() ignores overloaded getters.

More info here:  http://www.devsbedevin.com/the-pains-of-building-android-react-native-from-source/